templates/pages/admin/form/form_section.html.twig

```diff
@@ -235,6 +238,7 @@
 
     </section>
 
+    {% if section %}
     <div data-glpi-form-editor-section-blocks class="mt-3">
```

Comment:
This is what breaks the tests, we still need the div to be rendered so you need to place your new condition right after it.

Reply:
OK, I do not see =why we should render an empty div, but if that fix tests; it's OK for me (PR goal is to fix Twig, not html rendering)